_includes/head.html

```diff
@@ -51,26 +51,4 @@
 	<meta name="theme-color" content="#042075">
 
 	<link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
-
-	<style type="text/css">
-		{% capture mastheadImagePath %}{{ site.baseurl }}img/{{ page.path | replace:'_posts','posts' | replace:'.html','' | replace:'.md','' }}{% endcapture %}
-
-		#masthead, footer::before {
-			background-image: url("{{ mastheadImagePath }}.jpg");
-		}
-
-		@media (min-width: 1368px) {
-			#masthead, footer::before {
-				background-image: url("{{ mastheadImagePath }}@2x.jpg"),
-								  url("{{ mastheadImagePath }}.jpg"); /* fallback */
-			}
-		}
-
-		@media (min-width: 2736px) {
-			#masthead, footer::before {
-				background-image: url("{{ mastheadImagePath }}@4x.jpg"),
```

Comment:
Why remove all this responsive stuff?
If you do, you also need to remove all the matching images.